buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -428,6 +428,9 @@ class BuildPlugin implements Plugin<Project>  {
 
             Test integrationTest = project.tasks.create('integrationTest', RestTestRunnerTask.class)
             integrationTest.dependsOn(itestJar)
+            integrationTest.doFirst {
+                integrationTest.systemProperty("es.hadoop.job.jar", itestJar.getArchiveFile().get().asFile.absolutePath)
+            }
```

Comment:
I could not get this to use a closure - It looks like Gradle was trying to serialize it, I'm assuming in order to cache the input values, but the serialization did not resolve the value into a string first.

Reply:
That is correct. Gradle essentially just calls `toString()` on these. One thing we've done in our build is just to create an anonymous object whose `toString()` implementation returns the value we want.